ghost/core/core/server/services/stats/ContentStatsService.js

```
@@ -176,13 +176,31 @@ class ContentStatsService {
 
         // Enrich the data with post titles or UrlService lookups
         const enrichedData = await Promise.all(data.map(async (item) => {
+            // Check URL existence using the URL service
+            let urlExists = false;
+            
+            if (this.urlService && item.pathname) {
+                try {
+                    // Check if URL service is ready
+                    if (this.urlService.hasFinished && this.urlService.hasFinished()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)